### PR TITLE
Add github workflow to build and push MongoDB Atlas tools image

### PR DESCRIPTION
**Pull request by shahpratikr**
## Change Overview

This PR adds automation to create mongodb-atlas image to ghcr.io registry.

## Pull request type

Please check the type of change your PR introduces:
- [ ] :construction: Work in Progress
- [ ] :rainbow: Refactoring (no functional changes, no api changes)
- [x] :hamster: Trivial/Minor
- [ ] :bug: Bugfix
- [ ] :sunflower: Feature
- [ ] :world_map: Documentation
- [ ] :robot: Test

## Test Plan

- [ ] :muscle: Manual
- [ ] :zap: Unit test
- [ ] :green_heart: E2E

### CODE REVIEW

**Comment by viveksinghggits** [comment]
@shahpratikr 
We were not planning to build the image and publish it using kanister right? Did we change that?

**Comment by shahpratikr** [comment]
> @shahpratikr We were not planning to build the image and publish it using kanister right? Did we change that?

We discussed that for E2E tests, we will need to build image. hence added this change to build atlas image as well

File: .github/workflows/atlas-image-build.yaml [hunk 1]
```diff
@@ -0,0 +1,62 @@
+name: build-mongodb-atlas-tools-image
+
+on:
+  workflow_dispatch:
+    inputs:
+      tag:
+        description: 'MongoDB Atlas image tag in the format x.x.x'
```
Comment (viveksinghggits) [inline .github/workflows/atlas-image-build.yaml:7]:
```suggestion
        description: 'MongoDB Atlas image tag in the format x.y.z'
```

File: .github/workflows/atlas-image-build.yaml [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+jobs:
+  check-files:
+  runs-on: ubuntu-latest
+    outputs:
+      changed: ${{ steps.changed-files.outputs.any_changed }}
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+      - uses: tj-actions/changed-files@v35
```
Comment (viveksinghggits) [inline .github/workflows/atlas-image-build.yaml:24]: Is this verified GH action? If not, is there a possibility to use a verified action here.

**Review by viveksinghggits — COMMENTED** [review]
LGTM overall. We can merge this once we decide that we really want this in E2E.

File: .github/workflows/atlas-image-build.yaml (hunk 2 above)
Comment (shahpratikr) [reply .github/workflows/atlas-image-build.yaml:24]: Yes, I have verified this when we added kanister-build-image workflow.

File: .github/workflows/atlas-image-build.yaml (hunk 1 above)
Comment (shahpratikr) [reply .github/workflows/atlas-image-build.yaml]: fixed

File: .github/workflows/atlas-image-build.yaml (hunk 2 above)
Comment (viveksinghggits) [reply .github/workflows/atlas-image-build.yaml:24]: what I meant is, is this GitHub Action verified in the GitHub Actions marketplace.

Comment (shahpratikr) [reply .github/workflows/atlas-image-build.yaml:24]: yes. https://github.com/marketplace/actions/changed-files#usage

**Pull request merged**